spec/models/project_mediaflux_spec.rb

```diff
@@ -27,14 +27,19 @@
     end
 
     context "it formats dates correctly for MediaFlux" do
+      # Mediaflux dates are in UTC and look like this "07-Feb-2024 21:48:01"
       let(:project) { FactoryBot.build(:project, created_on: "2024-02-22T13:57:19-05:00", updated_on: "2024-02-26T13:57:19-05:00") }
-      it "formats created_on as expected" do
+      it "formats created_on and changes timezone to UTC as expected" do
         project_values = ProjectMediaflux.project_values(project: project)
-        expect(project_values[:created_on]).to eq "22-FEB-2024 13:57:19"
+        expect(project_values[:created_on]).to eq "22-FEB-2024 18:57:19"
       end
       it "formats updated_on as expected" do
         project_values = ProjectMediaflux.project_values(project: project)
-        expect(project_values[:updated_on]).to eq "26-FEB-2024 13:57:19"
+        expect(project_values[:updated_on]).to eq "26-FEB-2024 18:57:19"
+      end
+      it "changes the timezone back to America/New_York when it gets it back from Mediaflux" do
+        parsed_from_mediaflux = Time.zone.parse("26-FEB-2024 18:57:19").in_time_zone("America/New_York").iso8601
+        expect(parsed_from_mediaflux).to eq "2024-02-26T13:57:19-05:00"
```

Comment:
I am not sure this test is testing any of our code, do we need it or am I not understanding it?